api/src/main/java/javax/enterprise/event/TransactionPhase.java

```diff
@@ -45,6 +49,15 @@
      * <p>
      * Identifies a before completion observer method, called during the before completion phase of the transaction.
      * </p>
+     * <p>
+     * It will also get invoked if
+     * <ul>
+     *     <li>there is no transaction in progress, or</li>
+     *     <li>the transaction already got rolled back, or</li>
```

Comment:
That wording would then include 'SystemException', right? 
I'd rather leave SystemException bombing out. This usually indicates a serious problem on the server. Not sure neither if my wording catches this well though.